server/oauth2_test.go

```diff
@@ -321,13 +315,32 @@ func TestParseAuthorizationRequest(t *testing.T) {
 			}
 
 			_, err := server.parseAuthorizationRequest(req)
-			if tc.wantErr {
-				require.Error(t, err)
-				if tc.exactError != nil {
-					require.Equal(t, tc.exactError, err)
-				}
-			} else {
+			if tc.expectedError == nil {
 				require.NoError(t, err)
```

Comment:
Could you rewrite this block using the same library for assertions? It is not so important, testify of testing, but sticking to one library for the test is important.